src/Microsoft.DotNet.Build.Tasks/Microsoft.DotNet.Build.Tasks.csproj

```diff
@@ -32,6 +32,8 @@
     <Compile Include="ParseTestCoverageInfo.cs" />
     <Compile Include="RemoveDuplicatesWithLastOneWinsPolicy.cs" />
     <Compile Include="PrereleaseResolveNuGetPackageAssets.cs" />
+    <Compile Include="UpdateProjectDependencies.cs" />
+    <Compile Include="ValidateProjectDependencyVersions.cs" />
```

Comment:
/cc @joperezr this will need to be merged with https://github.com/dotnet/buildtools/pull/342